application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -64,69 +64,65 @@
  *   copy-on-write and shares a lot of objects with any other TransitLayer instances.
  */
 public interface TransitService {
-  Collection<String> getFeedIds();
+  Collection<String> listFeedIds();
 
-  Collection<Agency> getAgencies();
-  Optional<Agency> findAgencyById(FeedScopedId id);
+  Collection<Agency> listAgencies();
+  Optional<Agency> findAgency(FeedScopedId id);
```

Comment:
Shouldn't this be `getAgency`?

Reply:
It returns an optional, so that's why it's `find`.